pegasus/sites.v3/code.org/public/yourschoolteststaging.haml

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Most of this file is copied from `./yourschool.haml` with a few lines changed (listed below)